.azure-pipelines/templates/coverage.yml

```
@@ -7,6 +7,7 @@ jobs:
   - job: Coverage
     displayName: Code Coverage
     container: $[ variables.defaultContainer ]
+    timeoutInMinutes: 13  # 8 + 5
```

This usually finishes in around 5 minutes, so can be lower:

```suggestion
    timeoutInMinutes: 10
```